server/api/ballots.py

```diff
@@ -199,6 +199,17 @@ def list_ballots_for_jurisdiction(
     jurisdiction: Jurisdiction,
     round: Round,
 ):
+    if request.args.get("count"):
+        count = (
+            SampledBallot.query.join(Batch)
+            .filter_by(jurisdiction_id=jurisdiction.id)
+            .join(SampledBallotDraw)
+            .filter_by(round_id=round.id)
+            .distinct(SampledBallot.id)
+            .count()
+        )
+        return jsonify({"count": count})
+
```

Comment:
I'm a little bit irked by returning different data entirely based on a query parameter, as opposed to adding a `/ballots-count` endpoint. I can live with it, just not quite what I would expect from that API signature.

Reply:
I think the two common ways I've seen to do this are:
1. Use a response envelope and paginate the request, so that you get back something like: `{ ballots: [...], page: 1, count: 1000 }` (this is what I was emulating, though without the pagination, I didn't know what to do with the `ballots` field, so I just omitted it, figuring we could move to this pattern if/when we paginate)
2. Putting pagination and count in headers

I've never really seen a REST pattern for getting a count without having pagination, have you?

Reply:
ah, I see, this is a prelude to pagination... OK, I'm more comfortable with that then!